felix/rules/static.go

```diff
@@ -1429,6 +1440,16 @@ func (r *DefaultRuleRenderer) StaticRawOutputChain(tcBypassMark uint32) *generic
 		// return here without the mark bit set if the interface wasn't one that
 		// we're policing.
 	}
+
+	// Ensure VXLAN UDP Flows are not tracked in conntrack.
+	if r.VXLANEnabled || r.VXLANEnabledV6 {
```

Comment:
```suggestion
	if (ipVersion == 4 && r.VXLANEnabled) || (ipVersion == 6 && r.VXLANEnabledV6) {
```

Reply:
(You'll need to pass the IP version down into this method)

Reply:
Thanks, done.